front/app/containers/Admin/formBuilder/components/FormActions/index.tsx

```diff
@@ -45,13 +52,21 @@ const FormActions = ({
   const { projectId } = useParams() as {
     projectId: string;
   };
-  const project = useProject({ projectId });
-
+  const [showDeleteModal, setShowDeleteModal] = useState(false);
   const submissionCount = useFormSubmissionCount({
     projectId,
     phaseId,
-    projectType: project?.attributes.process_type,
   });
+  const closeModal = () => {
+    setShowDeleteModal(false);
+  };
+  const openModal = () => {
+    setShowDeleteModal(true);
+  };
+  const deleteResults = async () => {
+    await deleteFormResults(projectId, phaseId);
+    closeModal();
+  };
```

Comment:
I haven't tested this functionally, but do we show deletion confirmation to a user? Or what happens if we get an error from the BE regarding deletion?

Reply:
Yes, we show a modal to the user to confirm that they want to delete the results. We are however not handling errors that happen on deletion. I find that error handling within the application generally needs some work

Reply:
Ok - yeah it was the error handling / deletion confirmation (e.g. "Your results have successfully been deleted" I was referring to :) If we don't have this in other places on the app then it would probably be strange to just have it here. Should we bring this up at the next FE code discussion meeting maybe?? Align on a best practice?

Reply:
Yeah, let's do that.